src/schedulers/unstaked/unstaked.service.ts

```diff
@@ -44,21 +44,23 @@ export class UnstakedService implements OnModuleInit {
       const listRequestService = createRequestService.body.hits.hits;
   
       for (const requestService of listRequestService) {
-        const timeWaitingUnstaked: string = requestService['_source']['request']['unstaked_at'];
+        const timeWaitingUnstaked: string | null = requestService['_source']['request']['unstaked_at'];
```

Comment:
Making things null is actually kind of dangerous, probably refrain from using it like this. Instead, if you are expecting either an empty string it would be better.

Then just change the condition to `timeWaitingUnstaked !== ""` if possible.